### DISCUSSION TITLE
License remove on fork

### DISCUSSION THREAD

**Discussion by Wabri** [community/community#24015] [New to GitHub]
<p>Hello,</p>
<p>Is it possible to remove the license on a fork project?</p>
<p>One of my repository has been fork and I noticed that the license is no more present.</p>
<p>This is my repository: <a href="https://github.com/Wabri/Fly-By-Wire" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Wabri/Fly-By-Wire</a><br>
This is the fork: <a href="https://github.com/Martinligabue/Fly-By-Wire" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Martinligabue/Fly-By-Wire</a></p>
<p>I’m not interested in creating problems for that user because it’s a repository for a small university project, but I’d like to understand what would happen for larger projects.</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="wabri" data-post="1" data-topic="165322">
<div class="title">
<div class="quote-controls"></div>
 wabri:</div>
<blockquote>
<p>One of my repository has been fork and I noticed that the license is no more present.</p>
</blockquote>
</aside>
<p>That smells like copyright infringement to me (as a non-lawyer), especially considering that they removed the attribution your license requires along with the license. So I assume you could file a DMCA takedown request and possibly sue them.</p>

**Comment by sisrfeng** [comment]
Any official answer from github?

**Comment by Goldiserv** [comment]
The original repo is MIT licensed. I may be wrong but this gives permission for people to change the license & commercialize the code upon forks no?

**Reply by airtower-luna** [reply]
Read the MIT license, it's pretty short. It requires keeping attribution and the license for any copied code intact. :slightly_smiling_face:

**Reply by Goldiserv** [reply]
Thanks for the comment, led me to do some research into the topic, 
I now agree with your point.

When it comes to suing though, will be tricky to prove any damages:
https://www.quora.com/If-someone-doesnt-put-my-MIT-license-in-a-code-base-that-is-copied-from-my-open-source-project-can-I-sue 

For commercial products, looks like they can just link to the MIT & similar licenses like discord does: https://discord.com/licenses
